js/samples/quickstart-csharp/Controllers/HomeController.cs

```diff
@@ -12,7 +12,20 @@ public class HomeController : Controller
         private readonly string ClientSecret; // Azure AD Application Service Principal password
         private readonly string Subdomain;    // Immersive Reader resource subdomain (resource 'Name' if the resource was created in the Azure portal, or 'CustomSubDomain' option if the resource was created with Azure CLI Powershell. Check the Azure portal for the subdomain on the Endpoint in the resource Overview page, for example, 'https://[SUBDOMAIN].cognitiveservices.azure.com/')
 
-        private IConfidentialClientApplication app { get; set; }
+        private IConfidentialClientApplication _app;
+        private IConfidentialClientApplication app
```

Comment:
Looks good. Would just rename property app to ConfidentialClientApplication

Reply:
renamed